test/e2e/httpproxy/fqdn_test.go

```diff
@@ -81,4 +83,99 @@ func testWildcardFQDN(namespace string) {
 		err := f.CreateHTTPProxy(p)
 		require.NotNil(t, err, "Expected invalid wildcard to be rejected.")
 	})
+
+	Specify("wildcard routing works", func() {
+		t := f.T()
+
+		f.Fixtures.Echo.Deploy(namespace, "echo-slash-default")
```

Comment:
nit: maybe name these with a little more informative names, just so the test cases below read a little easier?